### PR TITLE
TASK: Refactor absolute namespace in neos fusion tests

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: TYPO3.TypoScript/Tests/Functional/TypoScriptObjects/Fixtures/Helper/UtilityHelper.php [hunk 1]
```diff
@@ -11,15 +11,18 @@
...
 {
     /**
      * @return void
-     * @throws \TYPO3\TypoScript\Exception
+     * @throws Exception
      */
     public function throwException()
     {
-        throw new \TYPO3\TypoScript\Exception('Just testing an exception', 1397118532);
+        throw new Exception('Just testing an exception', 1397118532);
```
Comment (daniellienert) [inline TYPO3.TypoScript/Tests/Functional/TypoScriptObjects/Fixtures/Helper/UtilityHelper.php]: Should be TypoScript\Exception

File: TYPO3.TypoScript/Tests/Functional/TypoScriptObjects/Fixtures/TypoScriptObjects/ThrowingImplementation.php [hunk 2]
```diff
@@ -27,7 +30,7 @@ protected function getShouldThrow()
     public function evaluate()
     {
         if ($this->getShouldThrow()) {
-            throw new \TYPO3\TypoScript\Exception('Just testing an exception', 1396557841);
+            throw new Exception('Just testing an exception', 1396557841);
```
Comment (daniellienert) [inline TYPO3.TypoScript/Tests/Functional/TypoScriptObjects/Fixtures/TypoScriptObjects/ThrowingImplementation.php]: Should be TypoScript\Exception

File: TYPO3.TypoScript/Tests/Unit/TypoScriptObjects/ResourceUriImplementationTest.php [hunk 3]
```diff
@@ -90,7 +92,7 @@ public function evaluateThrowsExceptionIfSpecifiedResourceIsInvalid()
      */
     public function evaluateReturnsResourceUriForAGivenResource()
     {
-        $validResource = $this->getMockBuilder('TYPO3\Flow\Resource\Resource')->disableOriginalConstructor()->getMock();
+        $validResource = $this->getMockBuilder(Resource::class)->disableOriginalConstructor()->getMock();
```
Comment (daniellienert) [inline TYPO3.TypoScript/Tests/Unit/TypoScriptObjects/ResourceUriImplementationTest.php]: Should be changed to PersistedResource

**Comment by johannessteu** [comment]
fixed

**Pull request merged**